pulpcore/pulpcore/app/viewsets/task.py

```diff
@@ -12,10 +13,19 @@
 
 
 class TaskFilter(filterset.FilterSet):
+    state_in_list = CharInFilter(name='state', lookup_expr='in')
+    worker = HyperlinkRelatedFilter(name='worker')
```

Comment:
I checked docs generated for `GET@/api/v3/tasks/` but I can't see neither type nor description of that field. I'm not able to figure out how to get that though (`label='text'` doesn't work) so I'm cool ;)

Reply:
I looked into this but the solutions seem painful. It would be nice to have an easy way to add descriptions for filters though and maybe something we could look into in the future.